ui/src/pages/backoffice/Series/SeriesSearch/SeriesList.js

```diff
@@ -100,6 +100,14 @@ class SeriesListEntry extends Component {
 
   render() {
     const { series } = this.props;
+    const identifiers = { issn: [], isbn: [] };
+    for (const identifier of series.metadata.identifiers) {
+      if (identifier.scheme === 'ISSN') {
```

Comment:
display depends on the vocabulary here, what if someone names it differently?

Reply:
Now only displaying the first identifier.